packages/core/src/sass/currency-select-radio.scss

```diff
@@ -9,7 +9,7 @@
         flex-wrap: wrap;
 
         @include desktop {
-            justify-content: center;
+            justify-content: flex-start;
```

Comment:
Should be centred for EU accounts if I remember correctly? As they only have a few currencies available,  could you check this with QA 🙏

Reply:
Sure :+1:

Reply:
Checked. It's just for fiat. We don't offer crypto to EU and they don't see `add or manage account` modal.